docs/securing-control-plane.md

```diff
@@ -0,0 +1,154 @@
+# Securing Control Plane
+
+All API communication between Antrea control plane components is encrypted with
+TLS. The TLS certificates that Antrea requires can be automatically generated.
+You can also provide your own certificates. This page explains the certificates
+that Antrea requires and how to configure and rotate them for Antrea.
+
+## Table of Contents
+
+- [What certificates are required by Antrea](#what-certificates-are-required-by-antrea)
+- [How certificates are used by Antrea](#how-certificates-are-used-by-antrea)
+- [Providing your own certificates](#providing-your-own-certificates)
+  - [Using kubectl](#using-kubectl)
+  - [Using cert-manager](#using-cert-manager)
+- [Certificate rotation](#certificate-rotation)
+
+## What certificates are required by Antrea
+
+Currently Antrea only requires a single server certificate for the
+antrea-controller API server endpoint, which is for the following communication:
+- The antrea-agents talks to the antrea-controller for fetching the computed
+ NetworkPolicies
+- The kube-aggregator (i.e. kube-apiserver) talks to the antrea-controller for
+ proxying the antctl's troubleshooting requests
+
+Antrea doesn't require client certificates for its own components as it
+delegates authentication and authorization to the Kubernetes API, using 
+Kubernetes [ServiceAccount token](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens)
```

Comment:
s/ServiceAccount token/ServiceAccount tokens

I think we have more than one token, depending on the component?

Reply:
done